packages/user-ui/src/components/input/mod.rs

```diff
@@ -26,3 +26,30 @@ pub fn InputBox(
         }
     }
 }
+
+#[component]
+pub fn Input(
+    #[props(default = None)] id: Option<String>,
+    #[props(default = "".to_string())] class: String,
+    #[props(default = "".to_string())] placeholder: String,
+    value: String,
+    onchange: EventHandler<String>,
+    #[props(extends = GlobalAttributes)] attributes: Vec<Attribute>,
+) -> Element {
+    rsx! {
+        input {
+            r#type: "text",
+            id,
+            class: format!(
+                "px-27 py-17 bg-box-gray rounded-lg text-[18px]/24 border-none focus:outline-primary  {}",
+                class,
+            ),
```

Comment:
```suggestion
            class: "px-27 py-17 bg-box-gray rounded-lg text-[18px]/24 border-none focus:outline-primary  {class}",
```
rsx 자체가 기본적인 string polutation 을 하기 때문에, 단순 대입은 굳이 format! 을 사용할 필요가 없어요.